pkgs/applications/misc/marktext/default.nix

```diff
@@ -2,16 +2,21 @@
 
 let
   pname = "marktext";
-  version = "v0.16.2";
-in
-appimageTools.wrapType2 rec {
+  version = "v0.16.3";
   name = "${pname}-${version}-binary";
```

Comment:
```suggestion
  version = "v0.16.3";
```
all appimage things now support pname+version.

Reply:
Cool, didn't know that. I would be happy to change this but given the recency of you introducing support for this ... that breaks backporting, no?

Reply:
> that breaks backporting

yeah, probably